bigtop-packages/src/charm/pig/layer-pig/actions/smoke-test

```diff
@@ -63,8 +62,8 @@ else:
 
 # The smoke tests analyze /etc/passwd. Successful runs will have an 'ubuntu'
 # passwd entry in the output.
+hookenv.action_set({'raw': output})
```

Comment:
@johnsca, proper benchmarks (using `charms.benchmark`) set benchmark-related bits like start/stop/composite/raw under the `meta` key.  Non-benchmark actions may still have helpful `raw` data, but I chose to display that as `results.raw` instead of creating a new `meta` key that only included `results.meta.raw`.